Combined sections for phases-5-6-7 (#270)
ietf-rats-wg/draft-ietf-rats-corim: draft-ietf-rats-corim.md

Appraisal policy inputs could result in Claims that augment the ACS.
For example, an Appraisel Policy for Evidence may specify that if all of a collection of subcomponents satisfy a particular quality metric, the top-level component also satisfies the quality metric.
The Verifier might generate an Endorsement ECT for the top-level component that asserts a quality metric.
Details about the policy applied may also augment the ACS.
An internal representation of policy detials, based on the policy ECT as described in {{sec-ir-policy}}, contain the environments affected by the policy with policy identifers as Claims.

Phase 7 is optional depending on implementation design.
Attestation Results rely on input from the ACS, but may not be interested in actual ACS entries.
For example, Attestation Results processing may map the ACS state to a generalized trustworthiness state such as {{-ar4si}}.
Generated Attestation Results Claims may be specific to a particular Relying Party.
Hence, the Verifier may need to maintain multiple Attestation Results contexts.

> An internal representation of Attestation Results as separate contexts, (see {{sec-ir-ars}}), ensures Relying Party specific processing doesn't modify the ACS, which is common to all Relying Parties.

Reviewer: ```suggestion
An internal representation of Attestation Results as separate contexts (see {{sec-ir-ars}}) ensures Relying Party–specific processing does not modify the ACS, which is common to all Relying Parties.
```
Parentheticals aren't also appositives.

I don't see how the sec-ir-ars reference elaborates the description of [ + ECT ] as "separate contexts". It's just another data definition that's equivalent to an ACS.

I don't think we've specified that the ACS is common to all relying parties either. There's nothing about the data definition that "ensures processing doesn't modify" something else. It's an organization technique that may help with that, but I don't know what this text is trying to say.

If all checks above have been performed successfully then the stateful environment matches.
If none of the candidate entries match the stateful environment entry then the stateful environment does not match.

## Examples for optional phases 5, 6, and 7 {#sec-phases567}

Phases 5, 6, and 7 are optional depending on implementation design.
Verifier implementations that apply consistency, integrity, or validity checks could be represented as Claims that augment the ACS or could be handled by application specific interfaces.
Processing appraisal policies may result in augmentation or modification of the ACS, but techniques for tracking the application of policies during appraisal need not result in ACS augmentation.
Additionally, the creation of Attestation Results is out-of-scope for this document, nevertheless internal staging may facilitate processing of Attestation Results.

Phase 5: Verifier Augmentation

> Claims related to Verifier applied consistency checks are asserted under the authority of the Verifier.

Reviewer 2: ```suggestion
Claims related to Verifier-applied consistency checks are asserted under the authority of the Verifier.
```

> For example, the `attest-key-triple-record` may contain a cryptographic key to which the Verifier applies certificate path construction and verification.

Reviewer 2: ```suggestion
For example, the `attest-key-triple-record` may contain a cryptographic key to which the Verifier applies certificate path construction and validation.
```

> The verification result may reveal an expired certificate.

Reviewer 2: ```suggestion
Validation may reveal an expired certificate.
```

The Verifier implementation might generate a certificate path validation exception that is handled externally, or it could generate a Claim that the certificate path is invalid.

Phase 6: Policy Augmentation

Appraisal policy inputs could result in Claims that augment the ACS.
For example, an Appraisal Policy for Evidence may specify that if all of a collection of subcomponents satisfy a particular quality metric, the top-level component also satisfies the quality metric.
The Verifier might generate an Endorsement ECT for the top-level component that asserts a quality metric.
Details about the policy applied may also augment the ACS.

> An internal representation of policy details, based on the policy ECT as described in {{sec-ir-policy}}, contain the environments affected by the policy with policy identifiers as Claims.

Reviewer 2: ```suggestion
An internal representation of policy details, based on the policy ECT as described in {{sec-ir-policy}}, contains the environments affected by the policy with policy identifiers as Claims.
```
